advance-exercises/src/components/01-CompoundComponents.js

```diff
@@ -16,70 +9,125 @@
     - Arrow left, arrow up should select the previous option
 */
 
-import React from 'react';
-import PropTypes from 'prop-types';
+import React from "react";
+import PropTypes from "prop-types";
 
 class RadioGroup extends React.Component {
   static propTypes = {
-    // defaultValue: PropTypes.string,                UN-COMMENT THIS LINE
-    children: PropTypes.shape().isRequired,
+    defaultValue: PropTypes.string,
+    children: PropTypes.shape().isRequired
   };
   render() {
-    return (
-      <div>{this.props.children}</div>
-    );
+    return <div>{this.props.children}</div>;
   }
 }
 
 class RadioOption extends React.Component {
   static propTypes = {
-    // value: PropTypes.string,                       UN-COMMENT THIS LINE
-    children: PropTypes.shape().isRequired,
+    value: PropTypes.string,
+    children: PropTypes.shape().isRequired
+  };
+  state = {
+    isSelected: false
+  };
+  handleClick = radio => {
```

Comment:
Adding an empty new line before a function starts, makes the code more readable. What do you think?

Reply:
Yup, sounds good. Thanks